kube-monitoring/charts/values.yaml

```diff
@@ -128,6 +128,19 @@ kubeMonitoring:
   kubeScheduler:
     enabled: false
 
+  ## Configuration for prometheus-node-exporter subchart
+  ##
+  prometheus-node-expoter:
+    enabled: true
+
+    relabelings:
+      - sourceLabels: [__meta_kubernetes_pod_node_name]
+        separator: ;
+        regex: ^(.*)$
+        targetLabel: nodename
```

Comment:
We are currently using the label `node` in all our clusters. So I would suggest we keep it consistency, otherwise dashboards will creak when switching datasources from a greenhouse monitoring stack to one of our existing ones

Reply:
Whatever we use it definitely should match the label that is used in kube-state-metrics so that its easy to create joins between node-exporter and cube-state-metrics timeseries.

Reply:
Fair enough. kube-state-metrics using a label `node`, so I will adopt this.